### ISSUE TITLE
Element Decompose does not update

### ISSUE THREAD

**Issue by scotttd** [mcneel/rhino.inside-revit#85]
If I have an object selected, expand Element decompose it works.  But, If I then choose a different object type with Geometric.Element, the Element Decompose does not update automatically on a change.

![image](https://user-images.githubusercontent.com/887876/70461788-47b52500-1a6e-11ea-9ffb-305f61f2e742.png)

**Comment by eirannejad** [comment]
Confirmed this issue still stands

![RihkJ9TlJB](https://user-images.githubusercontent.com/8197916/73398288-08e76280-429a-11ea-8863-0dd81af9f646.gif)

**Comment by kike-garbo** [comment]
It's supposed to work the way @eirannejad shows in the video, the user needs to refresh it.
The component starts empty without any parameter.
After feeding it with some elements the user can update the outputs with:
- All parameters that come from the input elements
- Or only the common parameters in all input elements.

Then the user may connect the output he decides to an other component.
After that can remove uconnected outputs from the component.

Do you really expect that the component rebuild itself based on the input?
What should it happen with the output wires, that may disapear, if you change the element type from one family to an other?
Imagine this in a more complex definition that some times feed this component with Walls and others with Columns!!
After some executions the component ouputs will be disconnected.
Destroying user definition.

**Comment by scotttd** [comment]
I can see that reconfiguring is a problem for the structure of a definition.  Would it help to have a note or warning off the top right that the parameters shown are not the complete list or that the current parameters?  Or a black label at the bottom that state common, all or partial parameters?  Just some way of knowing there is more available if the component is refreshed?  I see that will a right click using All parameters or Common Parameters can be used as a refresh.

**Comment by eirannejad** [comment]
I agree. The component can notify the user that the source element has changed.
